### ISSUE TITLE
MQTT connection

### ISSUE THREAD

**Issue by technic-home** [eifel-tech/ioBroker.syrconnect#1]
Is there a chance to get a MQTT connection with the syr products like mentioned here: https://www.loxwiki.eu/display/LOX/Syr+Safe+Tech
It seems that MQTT is integrated in the syr devices

**Comment by eifel-tech** [comment]
I think, there are only a few devices that support MQTT perhaps only one: Syr Safe Tec. I didn't find any other device where MQTT is listed in there datasheet. Most other ones are communicating with Syr Cloud. 
If you want to connect Syr Safe Tec with this protocol, please try to use mqtt adapter: https://github.com/ioBroker/ioBroker.mqtt

**Comment by technic-home** [comment]
Thank you for this reply and your adapter. I will ask Syr if they will add an API in future for automation systems. My hope is low, but I will give it a try. :-)

I will give you a feedback if I get a positive answer.

 
 

Gesendet: Montag, 26. April 2021 um 21:23 Uhr
Von: "eifel-tech" ***@***.***>
An: "eifel-tech/ioBroker.syrconnect" ***@***.***>
Cc: "technic-home" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [eifel-tech/ioBroker.syrconnect] MQTT connection (#1)


 

I think, there are only a few devices that support MQTT perhaps only one: Syr Safe Tec. I didn't find any other device where MQTT is listed in there datasheet. Most other ones are communicating with Syr Cloud.
If you want to connect Syr Safe Tec with this protocol, please try to use mqtt adapter: https://github.com/ioBroker/ioBroker.mqtt

**Comment by Richard-Schaller** [comment]
I'm also not aware of any API exposed by the SYR LEX Plus devices.

But based on the ideas from this project (ioBroker.syrconnect) I created [syrlex2mqtt](https://github.com/Richard-Schaller/syrlex2mqtt) which also simulates the SYR Connect Cloud and exposes the different sensors and configurations of a SYR LEX Plus 10 Connect device via MQTT. I then created a Home Assistant Add-on using syrlex2mqtt (see [addon-syrlex2mqtt](https://github.com/Richard-Schaller/addon-syrlex2mqtt)) making the integration as easy as possible.

@eifel-tech: I further reverse engineered and documented the [Syr Connect Protocol](https://github.com/Richard-Schaller/syrlex2mqtt/blob/main/doc/syrconnect-protocol.md). Maybe you find it useful for further extending the functionality of ioBroker.syrconnect.
